icons/jenkins/jenkins-original.svg

```diff
@@ -1 +1 @@
5.75-1.25.325-3.725.5-4.925.375zm4.5-6.2c-.075-.075-1.025-.3-2.1-.525-2.375-.45-5.1-.225-5.1.425 0 .125 1.65.225 3.675.225 2 0 3.6-.05 3.525-.125zm-.95-3.375c.425-.25-1.525-.6-3.375-.625-1.3 0-2.875.5-2.875.95 0 .275 5.65 0 6.25-.325zm-4.975-2a55.39 55.39 0 0 1 2.675-1.4c.725-.35 1.275-.7 1.225-.75-.05-.075-.925.05-1.95.25C96.7 91.975 94 93.425 94 94c0 .475 1.2.25 2.525-.5zm-8.65 9.325c-.075-.8-.425-3.2-.775-5.325-.325-2.125-.6-4.175-.6-4.55 0-.675.05-.7 1.275-.7 1.25 0 1.25 0 1.75 1.125.725 1.65 1.2 4.1 1.375 7.275l.175 2.75-1.325.425c-.725.225-1.4.425-1.5.425-.1 0-.275-.65-.375-1.425zM78.95 86c-.5-1.025-1.475-3.2-2.2-4.8-.7-1.6-1.8-3.675-2.4-4.6-.6-.925-1.1-1.725-1.1-1.8 0-.35.825-1 1.125-.9.175.075.725-.05 1.25-.25 1.35-.575 2.1-.5 3.375.35.625.4 1.375.75 1.7.75.55 0 .55.025.425 2.825-.2 3.925-.625 7.675-1 9.1l-.325 1.2zm.825-4.4c.425-.525 0-1.225-.775-1.225s-1.2.7-.775 1.225c.2.225.525.4.775.4s.575-.175.775-.4zm-.925-4.4c.325-.525-.05-1.275-.65-1.4-.4-.075-1.2.575-1.2.95s.8 1.025 1.2.95c.2-.05.5-.275.65-.5zM55.8 67c-2.525-.9-4.1-2.075-3.675-2.75.075-.125.5-.075.95.125.425.175 1.85.425 3.125.575 1.275.125 2.375.275 2.425.35.05.05-.025.575-.175 1.15-.3 1.225-.6 1.3-2.65.55zm-4.975-5.25c-1.15-.675-1.025-.85.125-.2 1.15.675 1.2.7.9.7-.125 0-.6-.225-1.025-.5zm-3.475-4.275c-.35-.7-.6-1.325-.55-1.375.05-.05.4.525.775 1.275.375.75.625 1.375.55 1.375-.075 0-.4-.575-.775-1.275zm-2.85-9.5c0-.2.325-.575.7-.825l.675-.475-.625.725c-.35.4-.65.775-.675.825-.05.05-.075-.05-.075-.25zm-8.925-10.35c0-.55.05-.75.1-.425.05.3.05.75 0 1-.075.225-.125-.025-.1-.575zm1.675-4.25c.475-.475.9-.875.975-.875.075 0-.25.4-.725.875s-.9.875-.975.875c-.075 0 .25-.4.725-.875zm6.325-.2c-.1-.175-.475-.525-.825-.75-.325-.225-.525-.425-.425-.425.35 0 1.375.775 1.525 1.15.175.45 0 .45-.275.025zm-3.125-1.35c.3-.05.75-.05 1 0 .225.075-.025.125-.575.1-.55 0-.75-.05-.425-.1zm-3.075-3.475c-.25-2.2 0-5.15.525-6.725.825-2.375 4.025-4.725 7.575-5.525 1-.225 1.85-.35 1.9-.3.075.05.15.35.175.675.075.5-.1.625-1.75 1.05-2.45.625-4.1 1.55-5.55 3.15-1.7 1.875-2.325 3.575-2.375 6.625-.025 1.35-.1 2.5-.175 2.6-.1.075-.225-.625-.325-1.55zm20.8-9.725c.3-.4.9-1.075 1.35-1.5.625-.55.55-.425-.25.5-1.475 1.675-1.7 1.875-1.1 1zM41.5 14.875c0-.375 1.225-1.975 2.425-3.2 2.325-2.35 5.675-4.4 9.55-5.775 1.275-.475 1.925-.6 2.125-.4.425.425-.1.8-3.25 2.325-2.6 1.25-4.75 2.775-6.6 4.675-.9.925-1.85 1.75-2.125 1.825-.275.075-.875.25-1.3.375-.45.15-.825.225-.825.175zm46.75-1.125c-.15-.275-.175-.5-.075-.5.125 0 .35.225.525.5.175.275.2.5.05.5-.125 0-.35-.225-.5-.5zM52.125 11c.95-.95 1.8-1.75 1.85-1.75.075 0-.65.8-1.6 1.75-.95.975-1.8 1.75-1.85 1.75-.075 0 .65-.775 1.6-1.75zm2.225-1.95c.2-.325 1.65-1.4 1.65-1.225 0 .075-.425.425-.925.825-.5.375-.825.575-.725.4ZM79.5 5.875c-.35-.2-.5-.35-.375-.35.15 0 .525.15.875.35.35.2.525.35.375.35-.125 0-.525-.15-.875-.35zm-19.5-.5C60 5.2 62.025 4.2 62.15 4.3c.05.075-.425.35-1.025.65-.625.3-1.125.5-1.125.425Zm7.7-2.3c.3-.05.8-.05 1.125 0 .3.05.05.1-.575.1s-.875-.05-.55-.1z"/></svg>
```

Comment:
Both the new and the old versions here are missing some shadows on the eyelids. Here's the official logo: https://www.jenkins.io/images/logos/jenkins/jenkins.svg
![](https://www.jenkins.io/images/logos/jenkins/jenkins.svg)

Which can be found here: https://www.jenkins.io/artwork/